### DISCUSSION TITLE
Understanding Database Egress Reset in Supabase: Impacts on Data Usage and Costs for Free Plan Users

### DISCUSSION THREAD

**Discussion by liwa-dev** [supabase/supabase#13921] [Questions]
i have a question regarding the database egress reset in Supabase, specifically on the free plan. When does the database egress reset, if at all? How does it impact data usage and costs? For instance, my data was around 570MB before entering April, but now it has increased to 1.29GB out of the 2GB limit. I have paused my project for over a week and I'm curious to know if the database egress resets every month. can anyone help me?

**Comment by GaryAustin1** [comment]
It is supposed to be monthly.  There are no charges for Free Plan, just at some point if you are exceeding egress you get email/warnings and eventually cut off (I assume).  The database size does look writing if you exceed it.

**Reply by liwa-dev** [reply]
Thanks for the information! So, the database egress reset is supposed to be monthly on the free plan, but if I exceed the egress limit, I may receive email warnings and eventually get cut off. Good to know that there are no charges for the Free Plan. However, I'm a bit concerned about my current database size which has increased to 1.29GB out of the 2GB limit. Since my project has been paused for over a week, I'm wondering if the egress limit will reset automatically. There are also about 7 days left for next month, so do you think the database egress will be reset? I am still new to the platform

**Reply by GaryAustin1** [reply]
I've never exceeded bandwidth so can't comment personally.  I've see a user go over the egress limit on free and nothing had happened when they asked.  There was no follow up so don't know how it worked out later.

I seriously doubt (and have never noticed) any billing type things reset based on pause/restore.  Seems that would allow infinite bandwidth if you could pause/restore anytime it got to high.
